src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
- [ ] As per the project requirements, you are required to delete all the boilerplate text, images, and styles that the CRA created. You might not need this bit right now

![image](https://user-images.githubusercontent.com/86318284/189194828-4803a7d7-47dc-4306-

Reply:
I resolved the issue, thanks again @Nemwel-Boniface